benchmark/pred_score.md

```diff
@@ -0,0 +1,34 @@
+| Dataset       | Python             | R        | size |  comment
+| ------------- |:-----------------: | ---------:|---: |-----: |
+| auto93        | 0.478885004573     | 0.007852035 | 82| initially same |
```

Comment:
What does initially same mean?

Reply:
means the tree was same for depth = 2 or 3, and later it was different